src/compiler/scala/tools/nsc/typechecker/PatternTypers.scala

```diff
@@ -85,20 +85,23 @@ trait PatternTypers {
       def resultType  = (fun.tpe memberType member).finalResultType
       def isEmptyType = resultOfIsEmpty(resultType)
       def isOkay      = (
-           resultType.isErroneous
+          resultType.isErroneous
```

Comment:
Is this an intellij setting? I recently saw retronym insert a space for alignment.

Reply:
Yikes, no... I was fiddling with this line and this is on accident. Thanks. Normally I'm the alignment stickler.

Reply:
OK, I wasn't try to stick either.

Reply:
Truly I appreciate it. I've been told that lack of alignment can make for slow memory access and getting kicked out of the local Pathfinder group.